web/src/compose_actions.ts

```diff
@@ -440,6 +441,7 @@ export let start = (raw_opts: ComposeActionsStartOpts): void => {
     // compose-box do not cover the last messages of the current stream
     // while writing a long message.
     resize.reset_compose_message_max_height();
+    compose_tooltips.initialize_compose_tooltips();
```

Comment:
Why is this called from `start`, rather than once from `ui_init`?

Reply:
Well, I see what you're doing architecturally with the save/destroy logic; this works functionally fine for the compose box. 

This PR also completely breaks the tooltips on these elements in the message-edit UI. So you'll need to do some reworking to address that.

Reply:
My bad for failing to test the message edit UI detail, I remember having that in my mind (https://github.com/zulip/zulip/pull/33331#discussion_r2031081949) but it got lost somewhere in between.